src/loaders.ts

```diff
@@ -121,7 +121,7 @@ export class MultiABILoader implements ABILoader {
                     return r;
                 }
             } catch (err: any) {
-                if (err.status === 404) continue;
+                if (err.cause?.status === 404) continue;
```

Comment:
HMM I could have sworn there was some loader that threw an error that had a status property, but maybe I'm wrong?

Maybe we need a test that goes over empty result conditions for each loader we have.

What do you think? Are you into it or should I do it? :)

Reply:
@k-thornton Do you remember under what condition you were able to trigger a 404 status on our loaders?

I'm trying to add a test for it but haven't been able to reproduce yet. https://github.com/shazow/whatsabi/pull/161/files#diff-2d7e35d8a3a306281e2352b54fe4ec41b741e4a6bfe9d1233670ddf569b6b4e4R230-R244

Reply:
Ah looks like it's just the new Blockscout loader when using loadABI, fixing it.